symphony-sample/packer-templates/scripts/files/os.repo

```diff
@@ -0,0 +1,4 @@
+[OS]
+name=os
+baseurl="http://mirrors.service.networklayer.com/centos/7/os/x86_64/"
```

Comment:
This would only work if the build environment is an IBMCloud hosted machine

Reply:
changed to http://mirror.centos.org/centos/7/os/x86_64/